### PR TITLE
Update: README

### PR DESCRIPTION
**Pull request by momo-ozawa**
Part of https://github.com/wordpress-mobile/WordPress-iOS/issues/16861

## Description 

This PR adds documentation for the MobilePayKit library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,118 @@
...
+```
+
+### Swift Package Manager
+
+To integrate MobilePayKit into your Xcode project via [Swift Package Manager](https://swift.org/package-manager/), add MobilePayKit as a dependency in `Package.swift`:
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/Automattic/MobilePay-Apple", .upToNextMajor(from: "1.0.0"))
+]
```
Comment (momo-ozawa) [inline README.md]: Should the tag be `0.0.1` instead?

File: README.md [hunk 2]
```diff
@@ -1 +1,118 @@
-# MobilePay-Apple
+# MobilePayKit
+
+Client library for making in-app purchases on iOS and macOS
```
Comment (leandroalonso) [inline README.md]:
```suggestion
Client library for making in-app purchases on iOS and macOS Automattic apps
```

File: README.md [hunk 3]
```diff
@@ -1 +1,118 @@
...
+
+
+## Introduction
+
+MobilePayKit is a client library for making in-app purchases. This project facilitates in-app purchases by coordinating requests to:
+
+- Apple's [StoreKit APIs](https://developer.apple.com/documentation/storekit/original_api_for_in-app_purchase)
+- Our WP.com in-app purchase APIs that communicate with the [MobilePay WooCommerce plugin](https://mobilepaymentsp2.wordpress.com/2021/07/14/how-is-this-all-going-to-work-anyways/)
+
+Realistically this library is only useful for Automattic-based projects, but the idea is to share what we've made.
```
Comment (leandroalonso) [inline README.md:13]: Did you add that in the case we make the repo public?

File: README.md [hunk 4]
```diff
@@ -1 +1,118 @@
...
+
+
+## Features
+
+- Support for **Consumable** in-app purchases (Non-Consumable, Auto‑Renewable Subscriptions / Non-Renewable Subscriptions are not supported yet)
```
Comment (leandroalonso) [inline README.md]: You may want to create a list like this: https://github.com/wordpress-mobile/MediaEditor-iOS#features

And then leave subscriptions and other stuff as separated non-checked items.

**Review by leandroalonso — COMMENTED** [review]
@momo-ozawa all good! Left just some minor suggestions. :)

File: README.md (hunk 3 above)
Comment (momo-ozawa) [reply README.md:13]: Yes! Got feedback from the platform team about making the repo public (p1629993816012300/1629993067.012000-slack-CC7L49W13)

File: README.md (hunk 2 above)
Comment (momo-ozawa) [reply README.md]: Fixed! 020adb9

File: README.md (hunk 4 above)
Comment (momo-ozawa) [reply README.md]: Fixed! fe5ccfd

**Comment by momo-ozawa** [comment]
### TODO once this PR gets approved / merged:

- [x] Create a version tag `0.0.1` for the Swift package
- [x] Make this repo public

**Review by leandroalonso — APPROVED** [review]

**Pull request merged**